xmtp_mls/src/groups/group_mutable_metadata.rs

```diff
@@ -126,16 +142,18 @@ impl GroupMutableMetadata {
                 .unwrap_or_else(|| DEFAULT_GROUP_PINNED_FRAME_URL.to_string()),
         );
 
-        if let Some(message_expiration_from_ms) = opts.message_expiration_from_ms {
+        if let Some(message_disappearing_settings) = opts.message_disappearing_settings {
+            println!(
```

Comment:
Probably want to convert this to a tracing log or remove it. Also no need to .clone() for a debug print.

Reply:
this left out, will remove it.